testsuite/integration/smoke/src/test/java/org/jboss/as/test/smoke/managedbean/SimpleProducerBean.java

```diff
@@ -1,42 +0,0 @@
-/*
```

Comment:
@stuartwdouglas This one shold probably not be removed, as org.jboss.as.test.smoke.managedbean.ManagedBeanTestCase fails